website/content/partials/known-issues/internal-error-namespace-missing-policy.mdx

```diff
@@ -9,4 +9,133 @@ This impacts all auth methods.
 - 1.14.4 and 1.14.5
 - 1.15.0 and 1.15.1
 
-A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
\ No newline at end of file
+A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
+
+### Workaround
+During authentication, Vault derives inherited policies based on the groups an
+entity belongs to. Vault returns an internal error when attaching the derived
+policy to a token when:
+
+1. the token belongs to a different namespace than the one handling
+   authentication, and
+2. the derived policy does not exist under the namespace.
+
+
+You can resolve the error by adding the policy to the relevant namespace or
+deleting the group policy mapping that uses the derived policy.
+
+As an example, consider the following userpass auth method failure. The error is
+due to the fact that Vault expects a group policy under the namespace that does 
+not exist.
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+# Failed login
+$ vault login -method=userpass username=user1 password=123
+Error authenticating: Error making API request.
+
+URL: PUT http://127.0.0.1:8200/v1/auth/userpass/login/user1
+Code: 500. Errors:
+
+* internal error
+```
+
+</CodeBlockConfig>
+
+To confirm the problem is a missing policy, start by identifying the relevant
+entity and group IDs:
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault read -format=json identity/entity/name/user1 | \
+  jq '{"entity_id": .data.id, "group_ids": .data.group_ids} '
+{
+  "entity_id": "420c82de-57c3-df2e-2ef6-0690073b1636",
+  "group_ids": [
+    "6cb152b7-955d-272b-4dcf-a2ed668ca1ea"
+  ]
+}
+```
+
+</CodeBlockConfig>
+
+Use the group ID to fetch the relevant policies for the group under the `ns1`
+namespace:
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault read -format=json -namespace=ns1 \
+  identity/group/id/6cb152b7-955d-272b-4dcf-a2ed668ca1ea | \
+  jq '.data.policies'
+[
+  "group_policy"
+]
+```
+
+</CodeBlockConfig>
+Now that we know Vault is looking for a policy called `group_policy`, we can
```

Comment:
```suggestion
</CodeBlockConfig>

Now that we know Vault is looking for a policy called `group_policy`, we can
```

The content parser sometimes gets tripped up when rendering the page if tag elements (like `<CodeBlockConfig>` ) are too close to other content. As a best practice, make sure to have a newline before and after any tag elements.